packages/protocol/src/routers/ConnextHandler.sol

```diff
@@ -195,13 +195,16 @@ contract ConnextHandler {
       revert ConnextHandler__executeFailed_tranferAlreadyExecuted(transferId, nonce);
     }
 
+
     SafeERC20.safeIncreaseAllowance(IERC20(txn.asset), address(connextRouter), txn.amount);
+    txn.executed = true;
+    _failedTxns[transferId][nonce] = txn;
```

Comment:
please address the changes in PR #616

Reply:
Hey Pedro, per our offline discussions I believe I was wrong on my point. 
In `_failedTxns[transferId][nonce]`, the `nonce` is used to access the specific position in the array of struct array  FailedTxn[] of the _failedTxns mapping.
`mapping(bytes32 => FailedTxn[]) private _failedTxns;`
Therefore it is okay as is.

Reply:
I only added clarifying notes for the `nonce` argument in the functions that have it. So that is not mis-interpreted.

Reply:
Makes sense :)